src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -269,6 +285,10 @@ private VariableInformation(TupleValue results, VariableInformation parent, stri
             {
                 VariableNodeType = NodeType.AnonymousUnion;
             }
+            else if (Name[0] == '*')
```

Comment:
>Name [](start = 21, length = 4)

I think you need `Name.Length >= 1 &&`

Reply:
Will be using `Name.Length **>** 1 &&` instead as the first character in `Name `must be an asterisk. Let me know if there are any cases where Name can just be "*" though. Thanks for pointing this out!

Reply:
I don't know if GDB would ever return a name of just '*'. I guess your way is better.